bs-react-native-next/src/apis/TimePickerAndroid.re

```diff
@@ -0,0 +1,29 @@
+type options;
+
+[@bs.obj]
+external options:
+  (
+    ~hour: int=?,
+    ~minute: int=?,
+    ~is24Hour: bool=?,
+    ~mode: [@bs.string] [ | `clock | `spinner | `default]=?,
+    unit
+  ) =>
+  options =
+  "";
+
+type response = {
+  .
+  action: string,
+  hour: int,
+  minute: int,
+};
```

Comment:
I am surprised that quote aren't necessary here 😮

Reply:
I'm more surprised that they weren't.

fixed

Reply:
Quotes **are** required.